build.sh

```diff
@@ -17,8 +17,4 @@ cd ..
 
 # Build DRT
 cd cedar-drt
-cargo build
-
-# Build inner fuzz crate
-cd fuzz && RUSTFLAGS="--cfg=fuzzing" cargo build
-cargo test
+cargo fuzz build
```

Comment:
should we still `cargo test`?

Reply:
yeah, looks like this is also used in CI, so it should stay